### ISSUE TITLE
Dancing Lights not configurable in v10

### ISSUE THREAD

**Issue by lupestro** [League-of-Foundry-Developers/torch#18]
@Airwrecker drew this to my attention. The properties for dim and bright lights in tokens were moved into their own substructure in v9. We fixed this everywhere except where we were setting up the Dancing Light tokens. So it is broken in the main stream.

It also turns out to be broken in the v10 stream. Not only are the properties of the tokens wrong, they aren't taking into account the settings provided from the light sources configuration. So this needs a fix in the v10 stream as well.

**Comment by lupestro** [comment]
I did a little further investigation. While the properties passed to do the update are the older ones, something in the system is doing the adjustment internally, and the dancing lights are coming up as 10 ft dim light, as specified, so as best I can tell there isn't a bug in this area in v9. 

However, I did just add the code to make the lighting for Dancing Lights configurable in v10 and this code will be in the next drop. While I was at it, in v10, I used the light, sight, and texture sub-structures so it reflects the delivered structure of a v10 token document.
